### PR TITLE
Labview review

### PR DESCRIPTION
**Pull request by katiemacaulay**
Added NVIDIA- CUDA installation, and carsim plugin.
Moved all home files to a separate folder

### CODE REVIEW

File: docs/LV_client/quick_start/LabVIEW_client_quick_start.md [hunk 1]
```diff
@@ -6,9 +6,19 @@
 
 2. Install Unreal Engine **4.23** from [here](https://www.unrealengine.com/en-US/).
 
-3. Download the monoDrive Simulator or monoDrive Simulator Editor from [here](https://www.monodrive.io/register).
+    - In Epic Games Launcher, go to Marketplace >> Browse. Search and download VehicleSim Dynamics, this is a free plugin. From Epic Games website, you may sign-in and download from [here](https://www.unrealengine.com/marketplace/en-US/product/carsim-vehicle-dynamics).
```
Comment (bradleynull) [inline docs/LV_client/quick_start/LabVIEW_client_quick_start.md:9]: Use of second person here. Suggested edit:
```suggestion
    - In Epic Games Launcher, and select Marketplace >> Browse. Search for "VehicleSim Dynamics" and download this free plugin. 
    - Alternatively, from Epic Games website, sign-in and download the plugin [here](https://www.unrealengine.com/marketplace/en-US/product/carsim-vehicle-dynamics).
```

File: docs/LV_client/quick_start/LabVIEW_client_quick_start.md [hunk 2]
```diff
@@ -6,9 +6,19 @@
...
 
-4. Install the monoDrive Client from the VI Package Manager. *Make sure you run the VI Package Manager as Administrator.*
+3. Download NVIDIA CUDA Toolkit 10.2 from [here](https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64).
+    - Choose your OS/ Version and how to install
```
Comment (bradleynull) [inline docs/LV_client/quick_start/LabVIEW_client_quick_start.md:12]:
```suggestion
    - Choose OS/ Version and how to install
```

File: docs/LV_client/quick_start/LabVIEW_client_quick_start.md [hunk 3]
```diff
@@ -6,9 +6,19 @@
...
+    - Example:
+         - Operating System: Windows
+         - Architecture: x84_64
+         - Version: 10
+         - Installer Type: exe(local)
+
+4. Download the monoDrive Simulator or monoDrive Simulator Editor from [here](https://www.monodrive.io/register).
+
+5. Install the monoDrive Client from the VI Package Manager. *Make sure you run the VI Package Manager as Administrator.*
```
Comment (bradleynull) [inline docs/LV_client/quick_start/LabVIEW_client_quick_start.md:21]:
```suggestion
5. **Open the VI Package Manger as Administrator** and install the monoDrive Client.
```

File: docs/LV_client/quick_start/LabVIEW_client_quick_start.md [hunk 4]
```diff
@@ -33,11 +43,11 @@
 
 
 ## Run the VehicleAI Simulator Editor
-Quick start instructions and details on how to generate project files in the VehicleAI Simulator Editor can be found in [scenario editor](Scenario_editor.md).
+Quick start instructions and details on how to generate project files in the VehicleAI Simulator Editor can be found in [scenario editor](../../Scenario_editor.md).
 
 1. Download Visual Studio from [here](https://visualstudio.microsoft.com/).
 
-    - You'll want to make sure that you have the following options enable, if you do not have Unreal Engine **4.23** already installed there is an option to install it here.
+    - You'll want to make sure that you have the following options enable.
```
Comment (bradleynull) [inline docs/LV_client/quick_start/LabVIEW_client_quick_start.md:50]:
```suggestion
    - Make sure the following options enabled:
```

File: docs/Nvidia.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# NVIDIA drivers
```
Comment (bradleynull) [inline docs/Nvidia.md:1]: Maybe I'm missing it, but it doesn't look like this page is actually used anywhere.

File: docs/Scenario_editor.md [hunk 6]
```diff
@@ -2,9 +2,21 @@
 
 ## Getting Started
 
-1. Download Visual Studio from [here](https://visualstudio.microsoft.com/)
+1. Install Unreal Engine **4.23** from [here](https://www.unrealengine.com/en-US/).
 
-    - You'll want to make sure that you have the following options enable, if you do not have Unreal Engine **4.23** already installed there is an option to install it here.
+    - In Epic Games Launcher, go to Marketplace >> Browse. Search and download VehicleSim Dynamics, this is a free plugin. From Epic Games website, you may sign-in and download from [here](https://www.unrealengine.com/marketplace/en-US/product/carsim-vehicle-dynamics).
```
Comment (bradleynull) [inline docs/Scenario_editor.md:7]: This is already mentioned in the LabView documentation (see above). Can we just link to this from there and not have to reiterate the same information? If not, see my suggested edits above on this verbiage.

File: docs/Scenario_editor.md [hunk 7]
```diff
@@ -13,15 +25,15 @@
...
 
-3. Open VehicleAI_Editor zip file and extract all files
+5. Open VehicleAI_Editor zip file and extract all files
 
     <div class="img_container">
     <img class='lg_img' src="https://github.com/monoDriveIO/documentation/raw/master/docs/LV_client/quick_start_img/sensor_editor_extract.png"/>
     </div>
 
-3. Go to your VehicleAI_Editor directory and find VehicleAI.uproject
+6. Go to your VehicleAI_Editor directory and find VehicleAI.uproject
```
Comment (bradleynull) [inline docs/Scenario_editor.md:36]: Numbering isn't actually important in markdown (the render will properly number the list no matter what number you have here). 

I usually just create a list of ones to make re-ordering easier (not that you have to do that here, just FYI):

```markdown
1. Item one
1. Item two
1. Item three
``` 

Renders as:

1. Item one
1. Item two
1. Item three

**Comment by katiemacaulay** [comment]
@bradleynull Great suggestions, added most of the changes. Decided to leave the instructions in both places though.

**Comment by bradleynull** [comment]
@katiemacaulay Looks good to go! Approved!

**Pull request merged**